client/my-sites/theme/main.jsx

```diff
@@ -451,6 +469,7 @@ const ThemeSheet = React.createClass( {
 		return (
 			<Main className="theme__sheet">
 				<QueryTheme themeId={ this.props.id } siteId={ siteIdOrWpcom } />
+				{ isJetpack && <QueryTheme themeId={ this.props.id } siteId="wporg" /> }
```

Comment:
Yeah, the first one fetches the theme object from the JP site, and the second one fetches the supplementary one from the WP.org API.